# NOTE: the following enumeration is defined in FaultDetector.h, but
# is not implemented!  The 'value' field is either 0 (no fault) or 1
# (hardware error).

# int32 None = 0
# int32 Low Voltage = 1
# int32 Instantaneous Capacity Loss = 2
# int32 Thermal Fault = 4

Header header
uint32 value